engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ConstraintValidatorContextImpl.java

```diff
@@ -181,7 +189,7 @@ private ConstraintViolationCreationContext getDefaultConstraintViolationCreation
 	private abstract class NodeBuilderBase {
 
 		protected final String messageTemplate;
-		protected boolean expressionLanguageEnabled;
+		protected ExpressionLanguageFeatureLevel expressionLanguageFeatureLevel = defaultCustomViolationExpressionLanguageFeatureLevel;
```

Comment:
Since EL may now be enabled by default depending on configuration, shouldn't `enableExpressionLanguage` be renamed to `expressionLanguageFeatureLevel`?
On second thought, it also feels weird that we can call `enableExpressionLanguage(NONE)`.

Reply:
I think I prefer it the way it is.